schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
-);
+create table animals(
+    id integer,
```

Comment:
- [ OPTIONAL ] I highly recommend to  to mark the id column as primary key because it is important to be unique since the id shouldn't be redundant or null 🙏🏻✅ but for that since you created the table already I recommend to use alter command if you want to assign it as primary key